apps/coscrad-frontend/src/components/AllEntities/AllEntities.tsx

```diff
@@ -2,51 +2,53 @@ import { useEffect, useState } from 'react';
 import { getConfig } from '../../config';
 import './AllEntities.module.scss';
 
-/* eslint-disable-next-line */
```

Comment:
Let's be careful to make sure we are all on the same page with prettier formatting on the front-end.